.github/workflows/jira-sync.yml

```diff
@@ -0,0 +1,13 @@
+name: Sync GitHub issues to Jira
+on: [issues, issue_comment]
+
+jobs:
+  sync-issues:
+    name: Sync issues to Jira
+    runs-on: ubuntu-latest
+    steps:
+      - uses: canonical/sync-issues-github-jira@v1
+        with:
+          webhook-url: ${{ secrets.JIRA_WEBHOOK_URL }}
+          component: 'S-Gaming'
```

Comment:
Since we're renaming all the jira components anyways, I'd consider doing that now and using the new name here, so you won't have to change it later :)

Reply:
I've just renamed this to Gaming